packages/tooltip/README.md

```diff
@@ -1,133 +1,186 @@
-<sp-tooltip open placement="top" variant="positive">
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit
-</sp-tooltip>
-<sp-tooltip open placement="top" variant="positive">
-    <sp-icon-checkmark-circle slot="icon" size="s"></sp-icon-checkmark-circle>
-    Label
-</sp-tooltip>
 <sp-tooltip open placement="top" variant="positive">
     <sp-icon-checkmark-circle slot="icon" size="s"></sp-icon-checkmark-circle>
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit
+    Quest completed!
 </sp-tooltip>
 ```
 
-### Negative
+</sp-tab-panel>
+<sp-tab value="negative">Negative</sp-tab>
+<sp-tab-panel value="negative">
 
-This is the negative a.k.a. error variant of Tooltip
+Use `variant="negative"` for error messages.
 
 ```html
-<sp-tooltip open placement="top" variant="negative">Label</sp-tooltip>
-<sp-tooltip open placement="top" variant="negative">
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit
-</sp-tooltip>
 <sp-tooltip open placement="top" variant="negative">
     <sp-icon-alert slot="icon" size="s"></sp-icon-alert>
-    Label
+    Quest failed!
 </sp-tooltip>
-<sp-tooltip open placement="top" variant="negative">
-    <sp-icon-alert slot="icon" size="s"></sp-icon-alert>
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit
+```
+
+</sp-tab-panel>
+</sp-tabs>
+
+### Behaviors
+
+#### Overlay
+
+By default, Tooltip provides styling without behavior.
+
+<sp-tabs selected="overlay-trigger" auto label="Overlay Behaviors">
+<sp-tab value="overlay-trigger">Overlay Trigger</sp-tab>
+<sp-tab-panel value="overlay-trigger">
+
+You must combine it with an [Overlay Trigger](https://opensource.adobe.com/spectrum-web-components/components/overlay-trigger/#%22hover%22-content-only) to manage its overlay behavior.
+
+```html
+<overlay-trigger triggered-by="hover">
+    <sp-button slot="trigger" variant="secondary">Hover me</sp-button>
+    <sp-tooltip slot="hover-content" placement="bottom">
+        Tooltip overlay triggered by hover
+    </sp-tooltip>
+</overlay-trigger>
+```
+
+</sp-tab-panel>
+<sp-tab value="self-managed">Self-managed</sp-tab>
+<sp-tab-panel value="self-managed">
+
+For simpler use cases, you can use the `self-managed` attribute which automatically binds the Tooltip to its first interactive ancestor element's focus/hover interactions:
+
+```html
+<sp-action-button>
+    Items
+    <sp-tooltip self-managed>Use items during battle.</sp-tooltip>
+</sp-action-button>
+```
+
+This is especially useful when inserting an intermediate `<overlay-trigger>` would interfere with parent/child relationships, such as between `<sp-action-group>` and `<sp-action-button>`.
+
+</sp-tab-panel>
+</sp-tabs>
+
+#### Delayed Opening
+
+A tooltip can be configured to delay its opening using the `delayed` attribute. This adds a warm-up period of 1000ms before showing the tooltip:
+
+```html
+<sp-tooltip self-managed delayed>
+    This tooltip will show after a delay
 </sp-tooltip>
 ```
+
+### Accessibility
+
```

Comment:
I would add something about how the content cannot be interactive (link, button, form) since the tooltip itself is not focusable or included in the tab order. And, that use of `aria-describedby` also flattens any semantics like headings, so the tooltip also shouldn't contain rich formatting. If interactive content or rich formatting is desired, then direct them to use a popover (or coachmark?) instead.